gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -80,13 +77,9 @@
  */
 @Slf4j
 public class MysqlMultiActiveLeaseArbiter implements MultiActiveLeaseArbiter {
-  /** `j.u.Function` variant for an operation that may @throw IOException or SQLException: preserves method signature checked exceptions */
-  @FunctionalInterface
-  protected interface CheckedFunction<T, R> {
-    R apply(T t) throws IOException, SQLException;
-  }
 
   protected final DataSource dataSource;
+  private final MySQLStoreUtils mySQLStoreUtils;
```

Comment:
pretty unintuitive name for an instance (generally 'utils' suggests a grab-bag of `static` functionality).  solely based on the name, I immediately wonder: what is this meant to be used for?  if it's a cohesive class, it ought to name itself accordingly.

Reply:
how about `DBStatementExecutor`?